enterprise/api/v1/mixins.py

```diff
@@ -84,7 +91,10 @@ def update_course_runs(self, course_runs, enterprise_customer, enterprise_contex
                 course_run=course_run,
```

Comment:
we should update the track selection url since we're still returning it in the api

Reply:
Actually, It was not mentioned in the story.

Reply:
My understanding of the story was any link that we publish relating to marketing a course or enrolling in a course, and the track selection page falls under that category. We want any traffic coming from enterprise's that are using our api to be tracked, so that includes this url.